changelog.d/3597.misc

```diff
@@ -0,0 +1 @@
+Add support for client_reader to handle more APIs
```

Comment:
I used .misc here on the assumption that this will automagically get merged with the changelog entry for https://github.com/matrix-org/synapse/pull/3555

Reply:
mmkay. it should probably be a .feature, then?

Reply:
Oh yes